### DISCUSSION TITLE
why do i have to press play 2 x times to play video?

### DISCUSSION THREAD

**Discussion by Szakul59** [TeamNewPipe/NewPipe#11529] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Please chane the play function.
I press on the video on the main screen and i expect to play that video automatically.
But instead i need to press play again in the next tab...

**Comment by ShareASmile** [comment]
Turn Auto Play to Always from Video Audio Settings.

Edited: After looking into issues anyone facing after AutoPlay to ON it may have been related to #10691
